src/Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG PYTHON_VERSION=3.8
+
+FROM python:${PYTHON_VERSION}
+
+WORKDIR /usr/src/app
+
+COPY . .
+
+RUN pip install --upgrade pip && \
+	pip install -r requirements.txt
+RUN python manage.py migrate
```

Comment:
Running 'python manage.py migrate' during the build process might not be ideal if the database is not available at build time. Consider running migrations as part of the container startup process instead.